Readme.md

```diff
@@ -1,28 +1,39 @@
 Gradle SonarQube Conventions Plugin [![Build Status](https://snap-ci.com/WeltN24/gradle-sonarqube-conventions-plugin/branch/master/build_image)](https://snap-ci.com/WeltN24/gradle-sonarqube-conventions-plugin/branch/master)
 =======================
 
-The plugin applies [SonarQube](http://www.sonarqube.org/) configuration to projects according our best practices at [welt](https://github.com/WeltN24).
+The plugin applies [SonarQube](http://www.sonarqube.org/) configuration and tasks to projects according our best practices at [welt](https://github.com/WeltN24).
 
 ## Usage
 
-See [plugin portal](https://plugins.gradle.org/plugin/de.weltn24.sonarqube).
+See [plugin portal](https://plugins.gradle.org/plugin/de.weltn24.sonarqube-conventions).
 
-## Conventions
+The plugin applies only conventions and best practices. Thus you have to configure the [sonarqube](https://plugins.gradle.org/plugin/org.sonarqube) in your project as well.
 
-### Plugins
+Example:
 
-The following standard Gradle plugins will be applied automatically:
+```
+buildscript {
+    repositories {
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
 
-+ [sonar-runner](https://docs.gradle.org/current/userguide/sonar_runner_plugin.html)
+    dependencies {
+        classpath(
+            'org.sonarqube.gradle:gradle-sonarqube-plugin:1.0',
+            'gradle.plugin.de.weltn24:sonarqube-conventions:2.0.0'            
+        )
+    }
+}
 
-### Tasks
+apply plugin: 'org.sonarqube'
+apply plugin: 'de.weltn24.sonarqube-conventions'
+``` 
 
-The plugin adds the follwoing tasks to the project:
+## Conventions
 
-* sonarRunner (standard tasks from [SonarRunner plugin](https://docs.gradle.org/current/userguide/sonar_runner_plugin.html))
-* sonarRunnerPreview
+### Tasks
```

Comment:
Please note that the tasks are crucial for the sonarqube runner (custom) configuration.